packages/apollo-gateway/src/__tests__/integration/networkRequests.test.ts

```diff
@@ -0,0 +1,36 @@
+import nock from 'nock';
+import { ApolloGateway } from '../..';
+
+import {
+  mockGetRawPartialSchema,
+  mockFetchStorageSecret,
+  mockGetCompositionConfigLink,
+  mockGetCompositionConfigs,
+  mockGetImplementingServices,
+  mockLocalhostSDLQuery,
+} from './nockMocks';
+
+it('Queries remote endpoints for their SDLs', async () => {
+  mockLocalhostSDLQuery();
+
+  let gateway = new ApolloGateway({
+    serviceList: [{ name: 'accounts', url: 'http://localhost:4001/graphql' }],
+  });
+  await gateway.load();
+  expect(nock.isDone()).toBeTruthy();
```

Comment:
what are we testing here?

Reply:
Nock mocks are one time use overides of node's HTTP request pipeline. This tests that each call we mocked was actually called

Reply:
Added tests for something more real.

Reply:
I like the idea of testing this, but having it as an `afterEach` seems more defensive?

Reply:
Added the `afterEach`